Runtimes/Core/CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ set(CMAKE_C_VISIBILITY_PRESET "hidden")
 set(CMAKE_CXX_VISIBILITY_PRESET "hidden")
 set(CMAKE_VISIBILITY_INLINES_HIDDEN YES)
 
+set(CMAKE_POSITION_INDEPENDENT_CODE YES)
```

Comment:
Uninformed question: does it make sense to configure that?
I stumbled into this [[PR for libcxxabi](https://reviews.llvm.org/D60049)](https://reviews.llvm.org/D60049) where they had the need to be able to disable that for WebAssembly, but I cannot gauge if that's something we may hit in this context.

Reply:
I think that we could always add that in the future if we need. In general, we should be building PIC because it is a security risk otherwise (PIC is needed to enable ASLR).